### PR TITLE
Signed-off-by: Christian Stark <cstark@cstark.muc.csb>

### PR DESCRIPTION
**Pull request by ch-stark**
Committer: Christian Stark <cstark@cstark.muc.csb>

- several improvements and new policies
- some policies must not be enforce

### CODE REVIEW

File: community/CM-Configuration-Management/policy-checkclusteroperator.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-checkclusteroperator
+  namespace:
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-checkclusteroperator.yaml]: Can you remove the empty `namespace` entry here.

File: community/CM-Configuration-Management/policy-configure-subscription-admin-hub.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# This policy looks for PolicyReport resources that contain failures.
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-configure-subscription-admin-hub.yaml]: You copied the wrong comments over here, can you remove or update these comments so they are specific to this policy.

**Review by gparvin — COMMENTED** [review]
You do have some overlap with the policy `community/AU-Audit-and-Accountability/policy-openshift-audit-logs-sample.yaml`.  Can you take a look at that one and make sure there's value in having yours and that one?  We could make changes to that one too if that's better.

I don't see README updates for everything either.  Can you double check that they will all have an update in the readme?

File: community/README.md [hunk 3]
```diff
@@ -68,6 +68,7 @@ Policy  | Description | Prerequisites
...
 [Example of labelling nodes of a cluster](./CM-Configuration-Management/policy-label-worker-nodes.yaml) | Use this policy to label nodes in your managed clusters. Notice you must know the name of the node or nodes to label.  | See the [OpenShift Documentation](https://docs.openshift.com/container-platform/4.5/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working) to learn more about labelling objects.
+[Example of a policy used to configure GitHub-Authentication](./CM-Configuration-Management/policy-deployment-sample.yaml) | Use this policy to log in to your OpenShift cluster with GitHub-Authentication. | See the OpenShift Documentation, [Configuring a GitHub or GitHub Enterprise identify provider](https://docs.openshift.com/container-platform/4.5/authentication/identity_providers/configuring-github-identity-provider.html) to learn more information.
```
Comment (gparvin) [inline community/README.md:74]: Change `policy-deployment-sample.yaml` to the right policy filename.

**Review by gparvin — COMMENTED** [review]
Still need some readme updates, let me know if you want to deliver this and I can go back and make readme updates or I could add updates to your PR too if you need me to help work on it

**Comment by ch-stark** [comment]
@gparvin made several improvements, update docu, hope it's ok

**Review by gparvin — APPROVED** [review]
Excellent policies!  Thanks!

**Pull request merged**